Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/ViewportInteractionImpl.cpp

```diff
@@ -58,4 +60,9 @@ namespace AtomToolsFramework
     {
         return m_deviceScalingFactorFn();
     }
+
+    void ViewportInteractionImpl::OnViewportDefaultViewChanged(AZ::RPI::ViewPtr view)
+    {
+        m_viewPtr = AZStd::move(view);
+    }
```

Comment:
I'm curious why use the move here? The ViewPtr type is a shared_ptr underneath the typedef so I'm unclear on the win.

Reply:
It's slightly faster as it avoids two atomic ref count operations (the increment for the acquire and the decrement when the lvalue parameter goes out of scope)